packages/commonjs/src/index.js

```diff
@@ -218,7 +221,11 @@ export default function commonjs(options = {}) {
             return `import { __moduleExports } from ${JSON.stringify(
               actualId
             )}; export default __moduleExports;`;
-          else if (esModulesWithoutDefaultExport.has(actualId))
+          /* return `export { default } from ${JSON.stringify(
+              actualId
+            )};`;*/ else if (
```

Comment:
Why did you comment this out? This will make sure that ES modules without a default export will go through the CJS helper in the catch-all case, which looks very wrong to me. But apparently this broke no test? In any case, I would definitely not leave it like this.

Reply:
I did not comment anything out here, it is one of my trials (to try and understand the point in all of this) that is commented out. A leftover that should not have been committed.